config/sidebar.yml

```diff
@@ -338,10 +338,10 @@ libraries:
     children:
 
```

Comment:
Without these changes the `Libraries > Auth0 SDK for Web` menu click was giving an error.